lib/board.rb

```diff
@@ -0,0 +1,50 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+
+
+  def reset!
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+
+  def initialize
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
```

Comment:
Consider using the `#reset!` method you already built inside `#initialize` to keep your code a little more DRY.  Also, by convention, list `#initialize` as the first method, right at the top under any `attr_accessors` and class variable declarations.  That way anyone looking at your code can immediately see what happens on instantiation of the class, without having to search for it.